Planet.xcodeproj/project.pbxproj

```diff
@@ -1180,6 +1180,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				COMBINE_HIDPI_IMAGES = YES;
 				DEVELOPMENT_ASSET_PATHS = "\"Planet/Preview Content\"";
+				DEVELOPMENT_TEAM = "";
```

Comment:
DEVELOPMENT_TEAM should not be included in the main `project.pbxproj` file.

We use a `local.xcconfig` file under Planet folder to manage it.